pandas/tests/test_algos.py

```diff
@@ -1797,13 +1797,13 @@ def test_too_many_ndims(self):
 
     @pytest.mark.single
     @pytest.mark.high_memory
```

Comment:
Is this still high memory?

Reply:
I think should be since still makes the big arrays when the test actually runs